createMedicCWBenchmark.sh

```diff
@@ -32,10 +32,14 @@ git push origin medic-cw-benchmarks
 # delete older benchmark
 cd benchmarks/setup/
 
-nameOfOldestBenchmark=$(ls | grep medic-cw- | sort | head -n 1)
-
-./stopBenchmarks.sh $nameOfOldestBenchmark
-./deleteBenchmark.sh $nameOfOldestBenchmark
+cw=$(date +%V)
+if [ $cw -gt 4 ]
+then
+  nameOfOldestBenchmark=$(ls | grep medic-cw- | sort | head -n 1)
+  ./deleteBenchmark.sh $nameOfOldestBenchmark
```

Comment:
👍  Nice idea to use `date +%V` for this

❓ You removed the stopBenchmarks.sh line entirely. I wasn't aware we could skip the stopBenchmark script when we want to delete things. So I want to ask you whether you considered this.

Reply:
Correct it is unnecessary. If you delete the namespace everything related with it will be deleted and stopped as well. It is like a cascade delete.